KMB/kmb_massload.py

```diff
@@ -153,6 +135,25 @@ def parser(dom, A):
     return A
 
 
+def process_tags(entry, dom, label, xml_tag):
+    """
+    Process tags of a given type.
+
+    :param entry: the dict of parsed data for the image
+    :param dom: the dom being analysed
+    :param label: the label under which the processed tags should be stored
+    :param xml_tag: the xml tag name to search for
+    """
+    entry[label] = []
+    elements = dom.getElementsByTagName(xml_tag)
+    for element in elements:
+        try:
+            entry[label].append(element.childNodes[0].data.strip())
+        except IndexError:
+            # Means data for this field was mising
```

Comment:
Have you seen this happen in a lot of cases, and for which fields? Is it something we might want to do something with, besides printing out a message?